.github/workflows/misspell.yml

```
@@ -0,0 +1,22 @@
+name: Misspelling check
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/misspell.yml"
+      - "**/*.adoc"
+      - "**/*.md"
+
+jobs:
+  misspell:
+    name: Misspelling check
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3.2.0
+      - uses: reviewdog/action-misspell@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          locale: "US"
```

This should fix the CI failures
```suggestion
          locale: "UK"
```

---

@omahs, this change was not enough to solve the `Misspelling Check` CI problems 😅, you will need to resolve each complaint of the checker, most of them are misspelled:

```
Error: [misspell] reported by reviewdog 🐶
"decentralized" is a misspelling of "decentralised"

Error: [misspell] reported by reviewdog 🐶
"amortizes" is a misspelling of "amortises"
...
```

you can find all here -> https://github.com/ChainSafe/gossamer/actions/runs/4068018700/jobs/7021954302

---

@EclesioMeloJunior almost all of them are correct US spellings, I changed "package" which was misspelled. Not sure if this workflow is useful if it creates issues like that.

---

@EclesioMeloJunior Perfect, just did that!